src/preload.py

```diff
@@ -53,14 +53,18 @@
 
 SUPERVISOR_REPOSITORY_RE = "^resin(playground)?/[a-z0-9]+-supervisor$"
 
+# 'sh' module '_truncate_exc' option:
+# http://amoffat.github.io/sh/sections/special_arguments.html#truncate-exc
+# https://github.com/amoffat/sh/blob/1.12.14/sh.py#L1151
+SH_OPTS = {"_truncate_exc": False}
```

Comment:
This `{"_truncate_exc": False}` option for the Python `sh` module prevents the "friendly" truncation of error output.